demo-app/src/POCMap.jsx

```diff
@@ -3,14 +3,66 @@ import { Box } from 'grommet';
 import { Grommet } from 'grommet-icons';
 import {
   Controls,
+  Cluster,
   Map,
   Marker,
   MarkerCluster,
   Pin,
 } from 'grommet-leaflet-core';
-
+import {
+  StatusCriticalSmall,
+  StatusGoodSmall,
+  StatusWarningSmall,
+  StatusUnknownSmall,
+} from 'grommet-icons';
 import { generateLocations, userLocation } from './utils/locations';
 import { ClusterPopup } from './ClusterPopup';
+import { getClusterStatus, getClusterSize } from './utils/status';
+
+const markerKinds = {
```

Comment:
from a demo-app project structure, it'd be nice to move this to a `themes` directory?

Reply:
Yes, I think so. I was running into some parsing issues with JSX in the js file when I made a `themes` directory. I should be able to sort that out and move towards something like this.

Reply:
Done.